### ISSUE TITLE
Lockscreen Widgets

### ISSUE THREAD

**Issue by nilsreichardt** [SharezoneApp/sharezone-app#1750]
![image](https://github.com/user-attachments/assets/71177d9e-ea4c-46de-8daf-e60cb15c5b12)

It would be great to have the timetable / the next lesson on the lockscreen as a widget. This makes it easier and quicker to look up the room for the lesson.

Related links:
* https://github.com/SharezoneApp/sharezone-app/issues/278
* https://github.com/ABausG/home_widget/issues/103

**Comment by deployor** [comment]
Yes! I’d absolutely love to see this as a feature! Apps like Noteapp already have it, and it’s so convenient to quickly check how much time is left in a lesson by just glancing at your phone or iPad. You could add circular widgets displaying the current lesson in the center or even one that shows the remaining time and the next lesson. A wide widget with details like the next lesson and room information would also be fantastic! There are so many useful things you could include!
